### ISSUE TITLE
List of values for filters

### ISSUE THREAD

**Issue by CharlesNepote** [rclement/datasette-dashboards#51]
List of values would be great for filters.

I hope the following example speaks by itself.
```yml
      filters:
        country:
          name: Country
          type: list
          values: world,be,es,fr,uk,us
        language:
          name: Language
          type: query
          # it has to be a unique column
          sql: select distinct language from this_table;
```

**Comment by rclement** [comment]
Thank your for your suggestion @CharlesNepote!

Indeed this type of filter is currently unsupported as the initial implementation is using a rather naive approach: simply being compatible with a subset of standard HTML input types (`text`, `date`, `numeric`):
https://github.com/rclement/datasette-dashboards/blob/b100f8de8cfd58866c6b7b18c64af51fa612c435/datasette_dashboards/templates/dashboard_view.html#L47

My initial thoughts about this would be to implement it using standard HTML `select` tag and `option` input type.

Would it be of interest to also be able to retrieve this value list from an SQL query? (I remember talking about this enhancement with someone but cannot find it anywhere)

**Comment by rclement** [comment]
@CharlesNepote Select filter type is now available (cf PR #53). Thank you for your suggestion!
